dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/HibernateTrackedEntityStore.java

```diff
@@ -362,12 +362,16 @@ private String getFromSubQuery(TrackedEntityQueryParams params, boolean isCountQ
             .append(" FROM trackedentity " + MAIN_QUERY_ALIAS + " ")
 
             // INNER JOIN on constraints
+            // filter={uid1} => Map.of(tea, List.of(null))
+            // filter={uid1}:eq:2 => Map.of(tea, List.of(new QueryFilter(EQ,"in")))
```

Comment:
I assume its

```suggestion
            // filter={uid1}:eq:2 => Map.of(tea, List.of(new QueryFilter(EQ,"2")))
```

right? Same for the comment below if that one is correct